BD-4400: Add details on managing consent for ad targeting for Audience Sync (#9344)
braze-inc/braze-docs: _docs/_partners/canvas_steps/overview.md

To learn more on how to comply with these Data Protection laws within the Braze platform, see [Data Protection Technical Assistance]({{site.baseurl}}/dp-technical-assistance/).

## Managing consent for ad targeting

> Managing consent for ad tracking or targeting of your end users is your responsibility as the advertiser.
> 
> To send ads to your end users, you must comply with all applicable laws and regulations as well as with the ad platform's policies and requirements. You should only use Braze to target and sync users where you have obtained their consent. 
> 
> To keep your audience lists in these ad platforms up-to-date and remove end users who have revoked their consent, you must set up a Canvas to remove users from these existing audience lists using an Audience Sync step.

Reviewer: ```suggestion
As the advertiser, it is your responsibility to manage consent for ad tracking or targeting of your users.

To send ads to your users, you must comply with all applicable laws and regulations, and the ad platform's policies and requirements. Only use Braze to target and sync users where you have obtained their consent. 

To keep your audience lists in these ad platforms up-to-date and remove users who have revoked their consent, set up a Canvas to remove users from these existing audience lists using an Audience Sync step.
```
Feel free to ignore, especially if this includes legal language!
